src/server/ua_server_internal.h

```diff
@@ -148,6 +148,9 @@ struct UA_Server {
     size_t namespacesSize;
     UA_String *namespaces;
 
+    /* Type Arrays */
+    const UA_DataTypeArray *typeArrays;
```

Comment:
Now we have a type array in the server config and internal.
Why both and how are they related?